drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -123,7 +123,7 @@
         </div>
         <div class="row">
           <label for="item-{{id}}-numerical-margin">
-            {{i18n "Margin ± (when a numerical value is required, values entered by students must not differ from the expected value by more than this margin; default is zero)"}}
+            {{i18n "Margin +/- (when a numerical value is required, values entered by students must not differ from the expected value by more than this margin; default is zero)"}}
```

Comment:
Already mentioned this on OC-1164 but for reference, https://github.com/edx/xblock-utils/pull/36 implements the fix for this issue.